### ISSUE TITLE
List libgcrypt as a requirement for OSX

### ISSUE THREAD

**Issue by johnhaley81** [nodegit/nodegit#1200]
Apple removed libgcrypt from OSX by default in Sierra. Developers now have to do a `brew install libgcrypt` to get NodeGit to build.

**Comment by maxkorp** [comment]
Why is libgcrypt required? Doesn't it use openssl? or did something change?

**Comment by maxkorp** [comment]
Yeah with a fresh Sierra install, without libgcrypt installed via brew, I am able to build.

**Comment by maxkorp** [comment]
Ohhhhh I bet i know, hold on

**Comment by jaywcjlove** [comment]
https://github.com/nodegit/nodegit/issues/1166

Seems to still have not solved the problem.

@johnhaley81
